### PR TITLE
Updating README with note to only use ppm v6 with this repo

### PR DESCRIPTION
**Pull request by shaun-od**
A readme update to state this repo will not work with pnpm v7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ Step package repositories use a _monorepo_ structure, which supports:
 
 A single step package repository is modelled on a volatility boundary - it groups together a set of steps and targets we reasonably expect to change together.
 
+> Note: Currently this will not work with pnpm v7+ so make sure you are running pnpm v6 with this repository.
```
Comment (geofflamrock) [inline README.md:42]: Is it worth putting any detail here on why only v6 is supported?

Comment (shaun-od) [reply README.md:42]: I didn't dig into why it happens so not sure what to put other than something along the lines of running pnpm commands with v7 causing a loop where the command is consistently rerun with parameters being appended. I can put that if you think that is useful.

Comment (geofflamrock) [reply README.md:42]: Yeah I think even just adding that as a simple "this is what happens if you use pnpm 7" would be helpful. If someone does accidentally use v7 then they might be able to work back from the symptoms to the cause and the fix.

Comment (shaun-od) [reply README.md]: Ok take a look at the change now and see if that is better 😃

**Review by geofflamrock — APPROVED** [review]
LGTM

**Pull request merged**